statics/media_selector.js

```diff
@@ -57,13 +57,17 @@ window.MediaSelector.prototype.init = function() {
         if (!data.item || !data.item.oid) {
             throw new Error('No oid in message from MediaServer page.');
         }
-        obj.onPick(data.item.oid, data.initial_pick);
+        obj.onPick(data.item.oid, data.item.thumb, data.initial_pick);
     }, false);
 };
 
-window.MediaSelector.prototype.onPick = function(oid) {
+window.MediaSelector.prototype.onPick = function(oid, thumburl) {
     var input = document.querySelector('#' + this.target + ' #id_mediaid');
     input.value = oid;
+    var input_thumb = document.querySelector('#' + this.target + ' input[name=mediaimg]');
```

Comment:
On our Moodle, there is no input matching `input[name=mediaimg]`, is this related to a custom Moodle theme ?

Reply:
Hi @sdiemer , this is actually a new thing in https://github.com/UbiCastTeam/moodle-atto_ubicast/blob/main/mod_form.php#L46 .

Reply:
Thank you for your response, I will merge this request.